### DISCUSSION TITLE
Please: How to modify the loading error when clicking on the licenses page

### DISCUSSION THREAD

**Discussion by karrynotme** [snipe/snipe-it#15572] [Q&A]
I know the correct page should be like this, but because I may have accidentally deleted the database content or there is a configuration problem, I cannot load this page correctly. Currently my error page is as shown in Figure. How can I modify it? Any advices will be appreciated!! 

![incorrectone](https://github.com/user-attachments/assets/042c9837-86d6-4de7-b10b-724849e2c97d)
![correct_one](https://github.com/user-attachments/assets/30eb12a6-8089-4860-9f2f-95dfa18472ac)

**Comment by snipe** [comment]
I think you need to remove the built-in aliases. https://snipe-it.readme.io/docs/xampp#remove-built-in-aliases

**Reply by karrynotme** [reply]
Thanks a lot!!! You are so amazing!!! I will try it later! ☺️
